chromium_src/chrome/browser/ui/views/toolbar/toolbar_view.cc

```diff
@@ -3,6 +3,9 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+#include "extensions/buildflags/buildflags.h"
+
+#if BUILDFLAG(ENABLE_EXTENSIONS)
```

Comment:
I think ToolbarView overriding is not just for extension?

Reply:
this particular set of headers could not be included without triggering an error because they include other extension headers. This is likely because chromium doesn't test non-default options so while they aren't strictly for extensions, they cannot be used without extensions being enabled